hs_collection_resource/templates/pages/collectionresource.html

```diff
@@ -419,6 +452,9 @@ <h4 class="modal-title" id="save-collection-warning-title">Deleted Resources</h4
                 removed_row_from_colletion.remove().draw(false);
             }
             $('#save-new').show();
+
+            // Rebind click event for new row
+            $("#collection-table-candidate td").click(onCollectionTableRowClick);
```

Comment:
@pkdash @zhiyuli I don't see connection of funding agency metadata update to changes in this collectionresource.html file, are these changes to collectionresource content html on resource landing page intended as part of this PR?